### PR TITLE
Database Performance Audit

### PR DESCRIPTION
**Pull request by dheerajsachdeva**
- Added More Data to Tables
- Analyze the performance
- Created Non Clustered Index to improve performance on queries

Screenshots:

image
image
image
image
image
image

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented; Ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe.

**Kindly remove or update the underlined:**
image
image

[NOT A REASON FOR INVALIDATION]
I'll also like to urge you to include details of the second contributor in the ReadMe. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by bobb-Rob — COMMENTED** [review]
Hi @dheerajsachdeva, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, and the README file has missing sections, Please ensure that you use all the sections as seen in the [README Template](https://github.com/microverseinc/readme-template/blob/master/README.md), and properly edited to suit your project. The only optional section is the FAQ section,...Please fix 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by dheerajsachdeva** [comment]
Done.

File: schema.sql [hunk 1]
```diff
@@ -58,4 +58,18 @@ create table visits (
...
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animals_id_desc ON owners(animals_id desc);
+
+
+CREATE INDEX email_asc ON owners(email ASC);
+
+
+CREATE INDEX vets_id_desc ON visits(vets_id asc);
+
```
Comment (algerina) [inline schema.sql:74]: - [ Optional]This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project.

As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:

` CREATE INDEX ON owners ((lower(email)));`

**Review by algerina — APPROVED** [review]
Hi @dheerajsachdeva  ,

Even if the execution times difference is very small in your case, I can clearly see that you had some fun with the explain analyze and I am sure that in the future you will know how to start the investigation for some slow DB queries :)

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

 ## Optional suggestions
Well... there is something else to say ;)
I left you some optional suggestions for improvements. Please check my comments below and consider them.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: schema.sql (hunk 1 above)
Comment (dheerajsachdeva) [reply schema.sql:74]: Ok. thanks

**Pull request merged**